docs/platform/developing/creating-tic.md

```diff
@@ -2,7 +2,7 @@
 
 This topic provides information on how to create a Tizen image with TIC.
 
-To use TIC, you have to set up the docker initially. For information on docker setup, see [Setting up Docker](../reference/setting-up-docker.md).
+To use TIC, you have to set up the docker initially. For information on docker setup, see [Set up docker](../reference/set-up-docker.md).
```

Comment:
The reference link is not showing up in the ToC.

Reply:
This link is working so I am resolving this conversation.